django_project/settings.py

```diff
@@ -3,15 +3,23 @@
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-# https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-SECRET_KEY
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/dev/howto/deployment/checklist/
+
+# https://docs.djangoproject.com/en/dev/ref/settings/#secret-key
+# SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = "django-insecure-0peo@#x9jur3!h$ryje!$879xww8y1y66jx!%*#ymhg&jkozs2"
 
 # https://docs.djangoproject.com/en/dev/ref/settings/#debug
+# SECURITY WARNING: don't run with debug turned on in production!
```

Comment:
Certainly!
But when WSVincent started DjangoX he wanted to keep it very basic: no env vars, no Docker, etc.

Since he added a Dockerfile, so maybe he mellowed a bit.

I for one, think an `.env` file would be simpler to navigate for a beginner than a huge `settings.py`.
If WSV is fine with it, I would happily prepare a merge request to get the relevant values using the `environs` package.

Reply:
Hi all! @quique is correct that I intentionally wanted to keep this package basic at first. There is an inevitable creep over time to complexity in all things. That said, it's probably a bad pattern to NOT HAVE environment variables and I don't think the bar is super high.

So @quique if you're open to it, I'd love to review a PR doing the basics using `environs`.

Reply:
Great !

I am going to work on it  :-)